pkgs/applications/version-management/smartgithg/default.nix

```diff
@@ -7,11 +7,11 @@
 
 stdenv.mkDerivation rec {
   name = "smartgithg-${version}";
-  version = "18_1_5";
+  version = "18_2_4";
```

Comment:
Can you do `s/_/\./g`?

Reply:
Amended: I have used `builtins.replaceStrings` to get underscores back for the URL. `replaceStrings` is included since nix 1.10

Reply:
Ah I didn't notice it was used in the url, but yeah I prefer it the way it's now, there's a bunch of other packages doing this too.